.github/workflows/release.yml

```diff
@@ -2,6 +2,14 @@ name: Release Artifacts
 
 on:
   workflow_dispatch:
+    inputs:
+      release-major-tag:
```

Comment:
How do we provide this input?

Reply:
Right now, we run the release build using the GitHub UI. With this change, GitHub will provide an option to specify this value. Unfortunately, I don't think GitHub Actions support "boolean" types. So you have to type in "true" or "false".